editor/components/AppBreadCrumb.tsx

```diff
@@ -19,17 +21,19 @@ const useCrumbs = (path: string): Crumb[] =>
       // we are at root
       return crumbs;
     }
+    // find the formatted route label
+    const pathPart = parts[0];
+    const route = routes.find((route) => route.path === pathPart.toLowerCase());
     crumbs.push({
-      // if we don't remove the query string, nextjs can hit a server/client mismatch on login
-      // todo: this can't be the right way to fix this
-      // todo: test if still an issue with app router
```

Comment:
this was indeed resolved completely by the app router. apparently the `usePathName` strips out the query string automatically